spinalcordtoolbox/deepseg/models.py

```diff
@@ -120,33 +122,105 @@
+                             'Dr. A. Althobathi of the University of Queensland, with the files consisting of T1w '
+                             'scans (and manual spinal cord segmentations) from 10 mice in total. The manual '
+                             'segmentations were then processed using the `sct_deepseg_gm` to obtain gray matter '
+                             'segmentations, which were then used in training. The dataset was comprised of both '
+                             'non-pathological (healthy) and pathological (diseased) mice.',
+         'url': 'https://github.com/ivadomed/mice_uqueensland_gm/',
          'models': ['mice_uqueensland_gm']},
     'seg_tumor_t2':
         {'description': 'Cord tumor segmentation on T2-weighted contrast',
+         'long_description': 'This segmentation model for T2w spinal tumor segmentation uses the UNet '
+                             'architecture, and was created with the `ivadomed` package. Training data consisted of '
+                             '380 pathological subjects in total: 120 with tumors of type Astrocytoma, 129 with '
+                             'Ependymoma, and 131 with Hemangioblastoma. This model is used in tandem with another '
+                             'model for specialized cord localisation of spinal cords with tumors '
+                             '(https://github.com/ivadomed/findcord_tumor).',
+         'url': 'https://github.com/sct-pipeline/tumor-segmentation',
          'models': ['findcord_tumor', 't2_tumor']},
     'seg_ms_sc_mp2rage':
         {'description': 'Cord segmentation on MP2RAGE in MS patients',
+         'long_description': 'This segmentation model for MP2RAGE spinal cord segmentation uses a Modified3DUNet '
+                             'architecture, and was created with the `ivadomed` package. Training data consisted of '
+                             'scans from 30 pathological (MS) patients, and the dataset included manual segmentations '
+                             'of MS lesions. This dataset was provided by the University of Basel.',
+         'url': 'https://github.com/ivadomed/model_seg_ms_mp2rage',
          'models': ['model_seg_ms_sc_mp2rage']},
     'seg_ms_lesion_mp2rage':
         {'description': 'MS lesion segmentation on cropped MP2RAGE data',
+         'long_description': 'This segmentation model for MP2RAGE MS lesion segmentation uses a Modified3DUNet '
+                             'architecture, and was created with the `ivadomed` package. Training data consisted of'
+                             '30 pathological (MS) patients that included manual segmentations of the spinal cord '
+                             'and MS lesions. The dataset was preprocessed to crop around the spinal cord prior to '
+                             'training. This dataset was provided by the University of Basel.',
+         'url': 'https://github.com/ivadomed/model_seg_ms_mp2rage',
```

Comment:
I've just now added missing `long_descriptions` for the MP2RAGE models that were added in SCT v5.5 (I only just noticed the descriptions were missing from this PR).

But, I feel like I'm missing details here (e.g. in-depth preprocessing steps). I tried to comb through the issues in https://github.com/ivadomed/model_seg_ms_mp2rage, but it might be better for @jcohenadad to fill in the gaps here, as it might be fresh(er) in your memory than it would be in mine.